src/libs/routes/authMiddleWare.ts

```diff
@@ -0,0 +1,24 @@
+import * as jwt from 'jsonwebtoken';
+import {hasPermission}  from '../permissions';
+import {permissions} from '../constants'
+export default ( module, permissionType )=> (req,res,next)=>{
+    try{
+    console.log("The config is: ",module,permissionType);
+    console.log("Header is: ", req.headers['authorization']);
+    const token= req.headers['authorization']
+    const decodeUser= jwt.verify(token,'qwertyuiopasdfghjklzxcvbnm123456')
+    console.log('user', decodeUser);
+    const a=decodeUser.Role;
+   hasPermission(permissions.getUsers,a,permissionType);
```

Comment:
authmiddleware is not working as per the task.

Reply:
resolved

Reply:
lint fixed